lib/resources/majors.tsx

```diff
@@ -2059,6 +2256,7 @@ const bsCS_Old: Major = {
             '<b>Intro Algorithms</b> <br /> EN.601.433 Intro Algorithms',
           required_credits: 3,
           criteria: 'EN.601.433[C]',
+          exclusive: ['N/A'],
```

Comment:
Do we need exclusive: ['N/A']? If there is no exclusivity, isn't it fine to just not specify the field because it is optional anyway?

Reply:
Ah I'm glad you brought it up. 
These are the possible values for `exclusive`: 
- undefined, always allows double count 
- ['N/A'], never allows double count 
- ['A', 'B'], it may overlap with distributions with the name field === 'A' or 'B'

In the case of `Algorithms`, 
- cannot overlap with `Upper Level CS Credits` or `CS Electives` => cannot be `undefined`
- can overlap with remaining 5 fine requirements => 
`exclusive: ['Gateway Computing: JAVA', 'Intermediate Programming', 'Data Structures', 'Computer System Fundamentals', 'Automata & Computation Theory', 'Team Requirement']`

Algo **never** satisfies these fine requirements. I give exclusive the value`['N/A']` so that no fine requirement is in the whitelist and we never bother running `checkRequirementSatisfied`. I do this only when I'm 100% certain a requirement never overlaps with another. 

Pros: 
- short circuiting logic, less function calls 
- saves us typing 

Cons: 
- inconsistent practice (sometimes I do this, other times I don't) 
- might introduce human error 

Hope that makes sense. I'm open to any suggestions. Lmk what you think. @aryamanshodhan @mattliu-mygit @kyang117